1/30/2025 5:02:16 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:02:16 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:02:16 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 5:02:17 PM:SignIn:btnLogIn_Click:test
1/30/2025 5:03:38 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:03:39 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 5:14:15 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
1/30/2025 5:15:30 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:15:35 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:16:14 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:16:15 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:16:23 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 5:16:25 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 5:16:25 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
30/01/2025 6:00:56 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
30/01/2025 6:01:23 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
1/30/2025 6:03:58 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:04:01 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 6:12:43 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
1/30/2025 6:19:58 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:19:59 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:20:09 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:20:13 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:20:31 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:20:33 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:20:56 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:20:59 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:21:49 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:21:54 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:30:32 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:30:39 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:51:30 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:51:31 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:57:40 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:57:40 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:58:58 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 6:59:02 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 7:03:41 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
1/30/2025 7:04:45 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 7:04:48 PM:Send_mail_APK:page_load:System.Data.SqlClient.SqlException (0x80131904): Connection Timeout Expired.  The timeout period elapsed while attempting to consume the pre-login handshake acknowledgement.  This could be because the pre-login handshake failed or the server was unable to respond back in time.  The duration spent while attempting to connect to this server was - [Pre-Login] initialization=28137; handshake=3;  ---> System.ComponentModel.Win32Exception (0x80004005): The wait operation timed out
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at StoreReportScheduler.send_mail() in D:\Working\POS\StoreReportScheduler.aspx.vb:line 47
   at StoreReportScheduler.Page_Load(Object sender, EventArgs e) in D:\Working\POS\StoreReportScheduler.aspx.vb:line 20
ClientConnectionId:48a8c715-e77f-4896-a28a-7d2ab6100ba4
Error Number:-2,State:0,Class:11
1/30/2025 7:04:52 PM:DBManager:connection_data:Object reference not set to an instance of an object.
30/01/2025 7:28:42 PM:Send_mail_APK:Send_mail_function:A transport-level error has occurred when receiving results from the server. (provider: TCP Provider, error: 0 - An existing connection was forcibly closed by the remote host.)
30/01/2025 8:11:52 PM:Send_mail_APK:Send_mail_function:A transport-level error has occurred when receiving results from the server. (provider: TCP Provider, error: 0 - The semaphore timeout period has expired.)
30/01/2025 8:12:09 PM:Send_mail_APK:Send_mail_function:A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)
1/30/2025 9:32:03 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 9:32:03 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 11:07:03 PM:DBManager:connection_data:Object reference not set to an instance of an object.
1/30/2025 11:07:20 PM:DBManager:connection_data:Object reference not set to an instance of an object.
